docs/releases.md

```diff
@@ -1,23 +1,39 @@
 ## Releases
 
-## Preparation
-
-To **prepare the extension for a new version**, you must:
-
-- update and commit the version in `package.json`
-- update and commit the `CHANGELOG.md`
-- tag, push, and merge to the `master` branch on GitHub
-
-## Packaging
-
-To **build a signed extension .xpi**, you must: commit and push to the
-`production` branch (ideally as a merge commit from `master`) on GitHub.
-
-## Extension Signing
+### Instructions
+
+1. Update "version" in package.json (and package-lock.json)
+2. Update "CHANGELOG.md" using `conventional-changelog`
```

Comment:
There should be a human spotcheck here, in case `conventional-changelog` missed something or added something unexpected.